_scripts/publishDockerImages.sh

```diff
@@ -2,4 +2,4 @@
 
 source "${BASH_SOURCE%/*}/common.bash"
 
-docker push govim/govim
+docker push -a govim/govim
```

Comment:
What's the rational behind this change?

Reply:
Apparently a change in behaviour of `cmd/docker`... at least it appears that way to me. I didn't dig too far, just fixed things up :)